README.md

```diff
@@ -1,4 +1,99 @@
 Sailjail permissions
 ====================
-
 Permissions for [Sailjail](https://github.com/sailfishos/sailjail/).
+These are [firejail](https://firejail.wordpress.com/) profile files with
+Sailjail specific metadata format.
+
+Permissions
+-----------
+Permissions that applications may use:
+
+- Accounts
+  - Using accounts, including editing them. Syncing accounts.
+- Ambience
+  - Set and edit ambiences.
+- AppLaunch
+  - Launching and stopping systemd services. This is usually needed for
+    background tasks.
+- ApplicationInstallation
+  - Installing and uninstalling applications.
+- Audio
+  - Playing audio, changing audio configuration and showing audio controls on
+    lockscreen.
+- Bluetooth
+  - Connecting to and using Bluetooth and NFC hardware.
+- Calendar
+  - Display and editing of calendar events.
+- Camera
+  - Access to camera hardware to take photos or video.
+- Contacts
+  - Display and editing of contacts data. Access to contact cards.
+- Documents
+  - Access to Documents directory.
+- Downloads
+  - Access to Downloads directory.
+- Email
+  - Reading and sending emails. Access to email attachments.
+- Internet
+  - Using data connection and connecting to internet.
+- Location
+  - Use GPS and low accuracy positioning.
```

Comment:
Is "low accuracy positioning" a common term?

Reply:
I'll remove "low accuracy"